Tests/Admin/BaseFieldDescriptionTest.php

```diff
@@ -194,13 +193,6 @@ public function testGetTranslationDomain()
         $this->assertSame('ExtensionDomain', $description->getTranslationDomain());
     }
 
-    public function testCamelize()
```

Comment:
This test should not be dropped but marked as legacy.

See: http://symfony.com/doc/current/components/phpunit_bridge.html#mark-tests-as-legacy